c7n/resources/rds.py

```diff
@@ -46,6 +46,7 @@
 import logging
 import re
 
+from pkg_resources import parse_version
```

Comment:
The reasoning was because StrictVersion doesn't work when the version number includes alphabet characters. LooseVersion could work, but is not as accurate as Strict.